packages/run-protocol/src/vaultFactory/storeUtils.js

```diff
@@ -6,15 +6,15 @@
 
 // XXX importing these that are declared to be used only for testing
 // until @agoric/store supports composite keys
-import { makeDecodeKey, makeEncodeKey } from '@agoric/store';
+import { makeDecodePassable, makeEncodePassable } from '@agoric/store';
 
 /** @typedef {[normalizedCollateralization: number, vaultId: VaultId]} CompositeKey */
 
 /**
  * @param {number} n
  */
-const numberToDBEntryKey = makeEncodeKey(/** @type {any} */ (null));
-const dbEntryKeyToNumber = makeDecodeKey(/** @type {any} */ (null));
+const numberToDBEntryKey = makeEncodePassable();
+const dbEntryKeyToNumber = makeDecodePassable();
```

Comment:
It isn't clear form the comment added above in liveslots what resulting type of passable is left from non remotable/promise/error.
It seems that it would still allow any primitive, or copy structure of primitive, right ? There is nothing to restrict it to number only?

Reply:
See #4896